### DISCUSSION TITLE
Nix package in the readme

### DISCUSSION THREAD

**Discussion by IvarWithoutBones** [Tyrrrz/DiscordChatExporter#922] [General]
Hello, I maintain this package in [nixpkgs](https://github.com/NixOS/nixpkgs/blob/4070161c5e2898c89ff834657745666025c5e7db/pkgs/tools/backup/discordchatexporter-cli/default.nix), the package repository for NixOS. I was wondering if you would be interested in adding it to the readme? I saw an AUR package is linked there as well.

**Comment by Tyrrrz** [comment]
Yes, sure, just submit a PR :)

**Comment by IvarWithoutBones** [comment]
Opened one: https://github.com/Tyrrrz/DiscordChatExporter/pull/925
